src/azure-cli/azure/cli/command_modules/acr/_help.py

```diff
@@ -1317,18 +1317,18 @@
 type: command
 short-summary: Retrieves information required to activate a connected registry.
 examples:
-  - name: Prints the values requiered to activate a connected registry in json format
+  - name: Sets http as the parent protocol, and prints the values required to activate a connected registry in json format
```

Comment:
Please the first person voice
```suggestion
  - name: Set http as the parent protocol, and prints the values required to activate a connected registry in json format
```